doc/release-notes.md

```diff
@@ -350,6 +350,20 @@ caching. A sample config for apache2 could look like:
 
     </VirtualHost>
 
+Asm script outputs now contain OP_CHECKLOCKTIMEVERIFY in place of OP_NOP2
```

Comment:
I'd suggest combining this with the other asm change related to signature flags, which is currently under "RPC: Low-level API changes".